### ISSUE TITLE
boxee plus streaming services

### ISSUE THREAD

**Issue by HolgO3000** [boxeehacks/boxeehack#551]
Hello,
I want to buy a boxee and install / uprade to boxee plus.
My question are as follows:
is it possible to integrate online video streaming services like 'maxdome' or 'cinema 2k' to the gui? Or do I have to use them via browser (if that is possible)?
I do not want to download movies, only watch streams.
I also have a Netgear ReadyNAS with Logitech Squeezebox server installed running. Is there a plugin / app for this or do I have to access it by common services?
Thanks in before (and for your great work)!
Best
HolgO

**Comment by jampeef** [comment]
think maxdome use Silverlight and it's not available for linux. Well there's moolight but that don't support DRM or Linux CE so i guess theres no way.

So you wanna play music on the boxee box from squeezebox server?

**Comment by HolgO3000** [comment]
Yes, I'm using the ReadyNAS with Logitech Squeezebox server for all my music (mp3). I have also Squeezebox Touch, Squeezebox Boombox, Squeezebox Radio and Squeezebox Duet standalone player in use.
You can access all files via Samsung TV allshare app  without problems.
Does anybody know if there is a plugin for boxee?

What about 'cinema 2k' as streaming service. Do I have to use the boxee internet browser or is there a plugin for the user interface?

**Comment by jampeef** [comment]
Don't know what cinema 2k uses.
I don't know of any squeezeplayer app for boxee, there's one for xbmc though xsqueeze or something.
